packages/platform-sdk-profiles/src/drivers/memory/wallets/wallet-transaction-service.test.ts

```diff
@@ -949,71 +986,72 @@ it("#restore", async () => {
 	expect(wallet.data().get(WalletData.SignedTransactions)).toContainKey(id);
 });
 
-it("sign a multisig transaction awaiting other signatures", async () => {
-	nock(/.+/)
-		.get("/transactions")
-		.query({
-			publicKey: "034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
-			state: "pending",
-		})
-		.reply(200, [require("../../../../test/fixtures/client/musig-transaction.json")])
-		.get("/transactions")
-		.query({
-			publicKey: "034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
-			state: "ready",
-		})
-		.reply(200, { data: [] })
-		.persist();
-
-	const id = await subject.signMultiSignature({
-		nonce: "1",
-		from: "DEMvpU4Qq6KvSzF3sRNjGCkm6Kj7cFfVaz",
-		data: {
-			publicKeys: [
-				"02edf966159de0013ca5b99371c5436e78f22df0d565eceee09feb977fe49cb910",
-				"0205d9bbe71c343ac9a6a83a4344fd404c3534fc7349827097d0835d160bc2b896",
-			],
-			min: 2,
-			senderPublicKey: "0205d9bbe71c343ac9a6a83a4344fd404c3534fc7349827097d0835d160bc2b896",
-		},
-		sign: {
-			mnemonics: ["this is a top secret passphrase 1", "this is a top secret passphrase 2"],
-			mnemonic: "this is a top secret passphrase 1",
-		},
-	});
-
-	expect(subject.transaction(id)).toBeDefined();
-	expect(subject.pending()).toContainKey(id);
-	expect(subject.waitingForOtherSignatures()).toContainKey(id);
-	expect(
-		subject.isAwaitingSignatureByPublicKey(
-			id,
-			"0205d9bbe71c343ac9a6a83a4344fd404c3534fc7349827097d0835d160bc2b896",
-		),
-	).toBeFalse();
-});
-
-it("should sync multisig transaction awaiting our signature", async () => {
-	nock(/.+/)
-		.get("/transactions")
-		.query({
-			publicKey: "034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
-			state: "pending",
-		})
-		.reply(200, [require("../../../../test/fixtures/client/multisig-transaction-awaiting-our.json")])
-		.get("/transactions")
-		.query({
-			publicKey: "034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
-			state: "ready",
-		})
-		.reply(200, [])
-		.persist();
-
-	const id = "a7245dcc720d3e133035cff04b4a14dbc0f8ff889c703c89c99f2f03e8f3c59d";
-
-	await subject.sync();
-	expect(subject.waitingForOurSignature()).toContainKey(id);
-});
+// @TODO: signing with all signatures at once is not really a real-world use-case
```

Comment:
This test signs a multi-signature transaction with all passphrases available at once. Not a real-world use-case, at least not a common one, but it might be worth it to introduce a `PrivateMultiSignatureSignatory` that can be used in case someone owns all private keys.